posthog/decorators.py

```diff
@@ -41,7 +41,14 @@ def wrapper(self, request) -> T:
             return f(self, request)
 
         filter = get_filter(request=request, team=team)
-        cache_key = generate_cache_key(f"{filter.toJSON()}_{team.pk}")
+        cache_key = f"{filter.toJSON()}_{team.pk}"
+        if request.data.get("invalidate_cache_key"):
+            cache_key += f"_{request.data['invalidate_cache_key']}"
+
+        if request.GET.get("invalidate_cache_key"):
+            cache_key += f"_{request.GET['invalidate_cache_key']}"
```

Comment:
Other places in the code say `cache_invalidation_key` – I suppose this here should also use that instead of `cache_invalidation_key`? Should there be a slightly more _integrating_ test to ensure there's no such mismatch – given that all of this logic sits in the backend?

Reply:
Yep fair, will add

Reply:
Done!